envision/web/src/components/app.js

```diff
@@ -61,26 +61,29 @@ function App({ client }) {
   const recorderRef = useRef(null);
   const { addToast } = useToasts();
   const history = useHistory();
+  const [totalSimIds, setTotalSimIds] = useState(0);
+
+  let getSimIdsLength = async () => {
+    let ids = await client.fetchSimulationIds();
+    setTotalSimIds(ids.length);
+  };
+  setInterval(getSimIdsLength, 3000);
```

Comment:
I believe this works better. The one remaining part of it is that we will need to fix the automatic navigation to a simulation page that occurs when the number of simulation IDs change but I also think we can leave this to a different PR.

Reply:
Hmm, I when I try, it doesn't navigates to simulation page when no. of sim IDs changes but remains on the simulation you were originally on.

Reply:
Oh, you meant when you are on "All Simulation" page. Yeah well that makes sense because for a new simulation ID it pushes it to the `history` and the server reconnects envision to that simulation if you are not connected to any simulation i.e. if you are on the multi-instance display.

Reply:
#961